Tests/PINRemoteImageTests.m

```diff
@@ -943,6 +944,51 @@ - (void)testQOS
     XCTAssert(dispatch_semaphore_wait(semaphore, [self timeout]) == 0, @"Semaphore timed out.");
 }
 
+- (void)testCacheControlSupport
```

Comment:
I think these would be (slightly) better tests if they actually checked the results of a second download instead of just seeing if the objects are still in the cache. Or perhaps both? PINRemoteImage will tell you how it got an object in result.resultType.

Reply:
OK.

Well it would be second & third downloads, to confirm that it got it from the cache and then did not.

Reply:
This change will make more obvious another limitation - The PINCache TTL implementation only applies to the disk cache. If you kept an app alive long enough, without memory warnings, you could still get results from the memory cache that the header would have suggested you shouldn't.

Reply:
Done